api/libretime_api/storage/management/commands/bulk_import.py

```diff
@@ -0,0 +1,157 @@
+import logging
+from pathlib import Path
+from typing import List, Optional
+
+import requests
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandParser
+from libretime_shared.files import compute_md5
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_ALLOWED_EXTENSIONS = [
+    ".flac",
+    ".m4a",
+    ".mp3",
+    ".ogg",
+    ".opus",
+]
```

Comment:
Maybe we should extend this list to something more realistic. Open for suggestions

Reply:
wav at least

Reply:
Ok, but we should highly recommend people not to use wav files as they don't store metadata.

Reply:
Maybe this can be moved to the configuration file actually, this will be used by the future django upload endpoint to filter unwanted files extension ?